Error running scorecard check

Maybe the vuln check is not working.

`GitHub token env var is not set. Please read https://github.com/ossf/scorecard#authentication" error="an error occurred while getting GitHub credentials`

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xf9b43d]

goroutine 79444 [running]:
github.com/ossf/scorecard/v4/checks/raw.Vulnerabilities(0xc00b0446c0)
	github.com/ossf/scorecard/v4@v4.5.0/checks/raw/vulnerabilities.go:35 +0xbd
github.com/ossf/scorecard/v4/checks.Vulnerabilities(0xc00b0446c0)
	github.com/ossf/scorecard/v4@v4.5.0/checks/vulnerabilities.go:40 +0x65
github.com/ossf/allstar/pkg/policies/scorecard.Scorecard.Check(0x0, {0x15d23f8, 0xc0005a4280}, 0xc0061d4a80, {0xc009ea3210, 0xd}, {0xc009ea3200, 0x8})
	github.com/ossf/allstar/pkg/policies/scorecard/scorecard.go:166 +0x658
github.com/ossf/allstar/pkg/enforce.runPoliciesReal({0x15d23f8, 0xc0005a4280}, 0x8d, {0xc009ea3210, 0xd}, {0xc009ea3200, 0x8}, 0x1)
	github.com/ossf/allstar/pkg/enforce/enforce.go:259 +0x1b8
github.com/ossf/allstar/pkg/enforce.runPoliciesOnInstRepos({0x15d23f8, 0xc0005a4280}, {0xc009c61fa0, 0x4, 0x3}, 0xc00051ef00)
	github.com/ossf/allstar/pkg/enforce/enforce.go:171 +0x143
github.com/ossf/allstar/pkg/enforce.EnforceAll.func1()
	github.com/ossf/allstar/pkg/enforce/enforce.go:131 +0x13e
golang.org/x/sync/errgroup.(*Group).Go.func1()
	golang.org/x/sync@v0.0.0-20220722155255-886fb9371eb4/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
	golang.org/x/sync@v0.0.0-20220722155255-886fb9371eb4/errgroup/errgroup.go:72 +0xa9

```

--- a/pkg/policies/scorecard/scorecard.go
+++ b/pkg/policies/scorecard/scorecard.go
@@ -26,6 +26,7 @@ import (
 	"github.com/ossf/allstar/pkg/scorecard"
 	"github.com/ossf/scorecard/v4/checker"
 	"github.com/ossf/scorecard/v4/checks"
+	"github.com/ossf/scorecard/v4/clients"
 
 	"github.com/google/go-github/v43/github"
 	"github.com/rs/zerolog/log"
@@ -157,10 +158,11 @@ func (b Scorecard) Check(ctx context.Context, c *github.Client, owner,
 
 		l := checker.NewLogger()
 		cr := &checker.CheckRequest{
-			Ctx:        ctx,
-			RepoClient: scc.ScRepoClient,
-			Repo:       scc.ScRepo,
-			Dlogger:    l,
+			Ctx:                   ctx,
+			RepoClient:            scc.ScRepoClient,
+			Repo:                  scc.ScRepo,
+			Dlogger:               l,
+			VulnerabilitiesClient: clients.DefaultVulnerabilitiesClient(),
 		}
 
 		res := checksAllChecks[n].Fn(cr)

